app/assets/stylesheets/layout_components/buttons/button-grouping.scss

```diff
@@ -97,6 +97,9 @@
       @extend .button-grouping__button;
       $button-background: $brand-red;
       background: $button-background;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
```

Comment:
This all looks good to me Sinead, I just wonder if we should add these styles straight to `button-grouping__button` instead of adding it to `button-grouping__button--red`.

I'm just thinking that the person who is using `button-grouping__button` won't be expecting positional changes to come along with `--red`.  

What do you think?